### PR TITLE
Migrate dashboards to dash endpoint

### PR DESCRIPTION
**Pull request by ChristineTChen**
### What does this PR do?

Validation is failing in CI on master due to some integrations having legacy payloads
### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by AlexandreYang — APPROVED** [review]

File: federatorai/assets/dashboards/application-overview.json [hunk 1]
```diff
@@ -1,224 +1,215 @@
...
-        },
-        {
-            "default": "*",
-            "prefix": "kube_stateful_set",
-            "name": "kube_stateful_set"
-        }
-    ],
+	"author_info": {
+        "name":"Federator.ai"
+    },
```
Comment (nmuesch) [inline federatorai/assets/dashboards/application-overview.json:6]:
```suggestion
	"author_name": "Federator.ai",
```

File: sigsci/manifest.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "assets": {
     "monitors": {},
     "dashboards": {
-      "Signal Sciences Overview": "assets/dashboards/overview.json"
+      "sigsci": "assets/dashboards/overview.json"
```
Comment (nmuesch) [inline sigsci/manifest.json:27]: 🙇 Thanks for fixing this to match what is currently synced

File: federatorai/assets/dashboards/cluster-overview.json [hunk 3]
```diff
@@ -1,378 +1,382 @@
...
-            "prefix": "host",
-            "name": "host"
-        },
-        {
-            "default": "24_hours",
-            "prefix": "prediction_window",
-            "name": "prediction_window"
-        }
-    ],
+	"author_name":"Federator.ai",
```
Comment (nmuesch) [inline federatorai/assets/dashboards/cluster-overview.json:4]: Indentation here looks off?

**Review by nmuesch — APPROVED** [review]
Indentation on the author_name fields look over indented, but otherwise LGTM!

**Pull request merged**